src/Components/Footer/FooterModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+const FooterModal = ({ isShowing, hide, title }) =>
+  isShowing &&
+  ReactDOM.createPortal(
```

Comment:
I don't et why you use a `React.createPortal` instead of a `usePortal`, it's much easier to use, isn't it ?